android/properties.gradle.kts

```
@@ -10,7 +11,8 @@ fun Project.readProperties(file: String): Properties {
       ->
       properties.load(reader)
     }
-  } else println("FILE_NOT_FOUND_EXCEPTION: File $file not found")
+  } else if(file.toString().contains(File.separator)) throw FileNotFoundException("\u001B[32mFile $file not found\u001B[0m")
```

couldn't you ref the file w/o a separator?

```suggestion
  } else if(!file.toString().empty?) throw FileNotFoundException("\u001B[32mFile $file not found\u001B[0m")
```

---

At the point we are loading the properties files we don't know yet which variant is being built. We can discuss some more with the team next week but this is the most optimal middle ground I found to resolve the issue I mentioned on your previous review feedback message.